src/browser/actions.js

```diff
@@ -81,7 +82,7 @@ define((require, exports, module) => {
     // once #239 is resolved.
     input: {value: '', isFocused: false},
     tabStrip: {isActive: false},
-    dashboard: {items: dashboardItems},
+    dashboard: initDashboard({items: dashboardItems}),
```

Comment:
Would rather not. Purpose of function is to be able to add/change data model fields in a central location. Analogous to `webviewer/actions.js` `open({})` function.